closed_session.go

```diff
@@ -89,7 +89,7 @@ func (s *closedLocalSession) destroy(error) {
 	})
 }
 
-func (s *closedLocalSession) getPerspective() protocol.Perspective {
+func (s *closedLocalSession) Perspective() Perspective {
```

Comment:
I still think this should be handled on the `http3` layer, not on the `quic` layer.